PWGHF/Tasks/taskLc.cxx

```diff
@@ -13,6 +13,7 @@
 /// \brief Λc± → p± K∓ π± analysis task
 /// \note Extended from taskD0
 ///
+/// \author Annalena Kalteyer <annalena.sophie.kalteyer@cern.ch>, GSI Darmstadt
```

Comment:
I would say it is clearer to add authors in chronological order.

Reply:
Sure. I moved the line to the end of the list. I was not sure what the convention was.